### ISSUE TITLE
Follow-up Check

### ISSUE THREAD

**Issue by martino-vic** [LoanpyDataHub/ronataswestoldturkic#3]
@LinguList I added you as a collaborator for a general follow-up check to see what's left to do to make this repository ready to add it to the lexibank community :)

**Comment by martino-vic** [comment]
Implemented all suggestions (changes can be viewed in the commits history), only pending thing is submitting the concept list to concepticon, where I have opened an issue, as indicated in the documentation there.

I'm sorry about the late reply, hope there's still the chance to write a blog post or similar about this some day

**Comment by LinguList** [comment]
Yep, this can definitely be done!

**Comment by martino-vic** [comment]
Update: all suggestions in https://github.com/martino-vic/rtbwestoldturkic/issues/3#issuecomment-1079646083 have been implemented. Now working on the blog-post, which can be accessed at this non-public repository: https://github.com/calc-project/calc-project/issues/91#issuecomment-1285592122

**Comment by martino-vic** [comment]
@LinguList I went through the repo once more, fixing some issues that came up recently. I think there might still be two open issues:

- #20
- I remember vaguely that there was something wrong with col "Loan" in [forms.csv](https://github.com/martino-vic/ronataswestoldturkic/blob/main/cldf/forms.csv). Might be this is a precedence and we have to brainstorm what the best convention would be. As I've understood the col "Loan" is supposed to tell whether the word in question is a loanword or not. That's why I've manually tagged all historical stages of Hungarian (H, OH, EAH, LAH) as True, and for West Old Turkic I went manually through the source pdf and set the value to True where the West Old Turkic word itself was borrowed (usually from a Mongolic language)
